benchmarks/benchmarks-schema.yaml

```diff
@@ -8,7 +8,7 @@ definitions:
     type: object
     properties:
       measureId:
-        description: measureId corresponds to a measure object's measureId
+        description: measureId corresponds to a measure object's measureId. For quality measures, measureId is the same as qualityId.
```

Comment:
I think this line is not necessary - all references to qualityId should be removed.